package.json

```diff
@@ -57,9 +59,10 @@
     "jest-cli": "^29.6.2",
     "marked": "^7.0.0",
     "rollup": "^3.27.0",
-    "semantic-release": "^21.0.7"
+    "semantic-release": "^21.0.7",
+    "tsd": "^0.28.1"
   },
   "dependencies": {
-    "smartypants": "^0.2.2"
+    "smartypants": "^0.2.2",
```

Comment:
```suggestion
    "smartypants": "^0.2.2"
```

Reply:
you will probably also need to do an `npm install` to update package-lock.json after moving dependencies

Reply:
Ah figured as much. I will update the PR this Thursday. Sorry for the inconvenience